benchmarks/ASTUnserializationBench.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Benchmarks;
+
+use GraphQL\Language\Parser;
+use Nuwave\Lighthouse\Schema\AST\DocumentAST;
+
+/**
+ * @BeforeMethods({"prepareSchema"})
+ */
+class ASTUnserializationBench
```

Comment:
Could we add phpbench into automated tests? Just so we can see the results in a PR, not necessarily make rules for how it should perform.

Reply:
I don't really want it in travis, as the benchmarks can get quite slow.

Reply:
I have no idea about how to run it async, however travis should just use the return code from the command. So if we make it not return an error code, that might work.

Just thinking that it could be useful for us to see the results of the benchmark whenever a user makes a new PR.

Reply:
Added it in as a job that is allowed to fail

Reply:
https://travis-ci.org/nuwave/lighthouse/jobs/530344137